media/js/externalpages/pocket/mobile-nav.es6.js

```diff
@@ -99,5 +98,17 @@ function handleKeyDown(e) {
     }
 }
 
-navOpenBtn.addEventListener('click', handleMenuOpen, false);
-navCloseBtn.addEventListener('click', handleMenuClose, false);
+const init = function () {
```

Comment:
separate init fn added to work with unit tests

Reply:
ah, good catch x2!